kolibri/plugins/audio_mp3_render/kolibri_plugin.py

```diff
@@ -1,33 +0,0 @@
-# -*- coding: utf-8 -*-
-"""
-Copy and modify this code for your own plugin.
-"""
-from __future__ import absolute_import, print_function, unicode_literals
-
-import logging
-
-from kolibri.plugins.base import KolibriFrontEndPluginBase
-from kolibri.plugins.hooks import CONTENT_RENDERER_ASYNC
-
-logger = logging.getLogger(__name__)
-
-
-class KolibriAudioMP3RenderFrontEnd(KolibriFrontEndPluginBase):
```

Comment:
I would rather these were migrated to the new system, rather than just deleted. They do serve a purpose.